:pencil: (proposal for spending community funds) (#709)
actualbudget/docs: blog/2025-05-30-spending-community-funds.md

---
title: Proposal for spending community funds
description: We're proposing a system to pay core maintainers for essential admin work funded by community donations. This 3-month trial won’t move forward without your input. If the community is against it, we won’t do it.
date: 2025-05-30T10:00
slug: spending-community-funds
tags: [announcement]
hide_table_of_contents: false
authors: MatissJanis
---

Our open source community has reached an exciting point: thanks to the incredible generosity of our donors, we now have a steady flow of funds coming in. First of all — thank you.

> This support gives us an opportunity to make the project more sustainable for everyone involved. And we want to start by addressing something too often overlooked in open source: the invisible labor that keeps everything running.

Reviewer: ```suggestion
This support allows us to make the project more sustainable for everyone involved. We want to start by addressing something that is often overlooked in open source: the invisible labour that keeps everything running.
```

We’re proposing a 3-month trial program to pay our core maintainers for the essential (but often unglamorous) work of:

- Reviewing pull requests
- Triage and categorize GitHub issues
- Preparing releases

> We think it’s time we recognized that labor financially — but before we move forward, we want your feedback.

Reviewer: In line 18 we have North American spelling _categorize_ - so _recognize_ it is:

```suggestion
We believe it’s time to recognize the financial value of labour — but before we move forward, we want to hear your feedback.
```

Reviewer 2: In that case do we need 'labor' instead of 'labour' too?

<!--truncate-->

## What We’re Proposing

> Starting July 1st, we’d like to set aside $1,000 per month from the project’s funds for a review stipend pool. This pool would be split between **core maintainers** based on the work they do each month, measured via a transparent points system.

Reviewer: ```suggestion
Starting July 1st, we’d like to set aside $1,000 per month from the project’s funds for a review stipend pool. This pool would be divided between **core maintainers** based on the work they do each month, measured via a transparent points system.
```

- Points are assigned based on the size of PRs reviewed (using lines of code as a proxy for effort).

> - Additional points are awarded for labeling and grooming issues, closing duplicates, and managing releases.

Reviewer: Labelling comes with double l.

```suggestion
- Additional points are awarded for labelling and grooming issues, closing duplicates, and managing releases.
```

> - The system is capped, and if the month is slow, the payout is reduced to $500.

Reviewer: I think this was a bit unclear what "slow" was referring to so I tried to clarify it.

```suggestion
- The system has a cap, and in months with less activity, the payout is reduced to $500.
```

> - Payments are handled via OpenCollective and only available to core maintainers.

Reviewer: ```suggestion
- Payments are handled via OpenCollective and are only available to core maintainers.
```

> This is designed to be lightweight, fair, and easy to administer — and it keeps our monthly costs well within our donation income.

Reviewer: ```suggestion
This system is designed to be lightweight, fair, and easy to administer — and it keeps our monthly costs well within our donation income.
```

## Why Start With Admin Work?

> Before we think about feature bounties or bug bounties, we need to acknowledge the day-to-day labor that enables any of those contributions to be merged and released. Paying for features while leaving critical admin work unpaid would be unsustainable — and unfair.

Reviewer: ```suggestion
Before we consider feature bounties or bug bounties, we need to acknowledge the day-to-day labour that enables these contributions to be merged and released. Paying for features while leaving critical administrative work unpaid would be unsustainable — and unfair.
```

Reviewer 2: labor vs labour

> This trial is the foundation for broader funding efforts down the line. But it starts with respecting the work already being done.

Reviewer: ```suggestion
This trial serves as the foundation for broader funding efforts in the future. However, it begins with respecting the work that is already being done.
```
